nodejs/eks/examples/examples_test.go

```diff
@@ -92,11 +92,10 @@ func Test_Examples(t *testing.T) {
 		}),
 	}
 
-	longTests := []integration.ProgramTestOptions{}
-	// longTests := shortTests
-	// for _, longTest := range longTests {
-	// 	longTest.RunUpdateTest = true
-	// }
+	longTests := shortTests
+	for _, longTest := range longTests {
+		longTest.RunUpdateTest = true
+	}
```

Comment:
Why not just modify all the tests upfront (like in `base`)?  Also `shortTests` vs. `longTests` naming is weird here - there is only one set of tests right?

Reply:
Or is the goal to run *both* with and without `RunUpdateTest`?  If so - that doesn't seem to be what this will accomplish.

Reply:
> Why not just modify all the tests upfront (like in base)? Also shortTests vs. longTests naming is weird here - there is only one set of tests right?

The `RunUpdateTest` is its own, singular test - that is, it does not test both the locally-linked && latest release tests; hence the need to set both sets of tests. I threw it in a for-loop to not have to redefine the same tests in `shortTests` given that the only diff was the `RunUpdateTest: true` setting.

It seems that redefining the same tests as `updateTests` separately from `shortTests` instead of using the for-loop will help clear things up. I'll make these changes.

Reply:
>  I threw it in a for-loop to not have to redefine the same tests in shortTests given that the only diff was the RunUpdateTest: true setting.

I don’t think that’s what the code as written accomplished - does it? If that is the intent - then I agree a for loop makes sense - but I think you would need to make a copy of the tests before mutating them - right?

Reply:
I totally overlooked that `shortTests` is a slice (pointer in Go) and that I was mutating it's value for `longTests` 🤦‍♂. Thank you for catching this!

I've changed it to use new, separate tests `updateTests`.

Reply:
FWIW - I was trying to just suggest that you copy the tests and overwrite this in the copies instead of having to write every test out twice.  But at least this now looks correct! :-).